app/src/main/java/org/oppia/app/player/state/StateFragmentPresenter.kt

```diff
@@ -354,6 +357,46 @@ class StateFragmentPresenter @Inject constructor(
     itemList.add(contentViewModel)
     stateAdapter.notifyDataSetChanged()
   }
+  private fun checkAndAddInteraction() {
+    if (currentEphemeralState.stateTypeCase.number == EphemeralState.PENDING_STATE_FIELD_NUMBER) {
+      when (currentEphemeralState.state.interaction.id) {
+        MULTIPLE_CHOICE_INPUT -> {
+          addMultipleChoiceInputItem()
+        }
+        ITEM_SELECT_INPUT -> {
+          addMultipleChoiceInputItem()
+        }
+      }
+    }
+  }
+
+  private fun addMultipleChoiceInputItem() {
+    val customizationArgsMap: Map<String, InteractionObject> =
+      currentEphemeralState.state.interaction.customizationArgsMap
+    val multipleChoiceInputInteractionViewModel = CustomizationArgsInteractionViewModel()
+    val allKeys: Set<String> = customizationArgsMap.keys
+
+    for (key in allKeys) {
+      logger.d(TAG_STATE_FRAGMENT, key)
+    }
+    when {
+      customizationArgsMap.contains("choices") -> {
```

Comment:
For this & the next when, suggest using an ``if ()`` instead since they are both single-cased. I think ``if``` will be slightly more comprehensible.

Reply:
Okay done.